src/dialogs/examples/basic/AdaptiveCardDialog.ts

```diff
@@ -0,0 +1,451 @@
+import * as builder from "botbuilder";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+
+export class AdaptiveCardDialog extends TriggerActionDialog {
+
+    private static async step1(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void>
+    {
+        // if replyToId is not null and if defined key in adaptive card data is present in incoming activity value
+        // to check if this request is from an adaptive card process adaptive card submit action values, it's a work around that will be cleaned up later
```

Comment:
Let's clean up this comment to say "Check for the property in the value set by the adaptive card submit action"

Below, don't check from replyToId and just have session.message.value && session.message.value.isFromAdaptiveCard

Reply:
Done.